src/main/java/scripts/base/TestBaseSetup.java

```diff
@@ -0,0 +1,123 @@
+package main.java.scripts.base;
```

Comment:
About extend:
Java only support extend 1 class, so should consider how to extend more convenience.